src/ui/Navbar.fs

```diff
@@ -4,67 +4,125 @@ open Feliz
-                                prop.style [ style.cursor "pointer" ]
-                            ]
+                        Link [
+                            prop.text "About Us"
+                            prop.onClick (fun _ -> Router.navigatePath "/about")
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
 
                     NavbarItem [
                         Link [
                             prop.text "Contact"
-                            color "foreground"
                             prop.onClick (fun _ -> Router.navigatePath "/contact")
-                            prop.style [ style.cursor "pointer" ]
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
                 ]
             ]
 
             NavbarContent [
                 justify "end"
+                prop.style [ length.px 40 |> style.gap ]
                 prop.children [
                     NavbarItem [
-                        Button [ color "primary"; prop.custom ("variant", "flat"); prop.text "Cart" ]
+                        img [
+                            prop.src <| import "default" "./assets/Cart.svg"
+                            prop.onClick (fun _ -> Router.navigatePath "/")
+                            prop.style [
+                                length.px 21 |> style.height
+                                style.cursor "pointer"
+                                (if isMenuOpen then 0 else 100) |> style.filter.invert
+                            ]
+                        ]
+                    ]
+
+                    NavbarMenuToggle [
+                        prop.ariaLabel <| if isMenuOpen then "Close menu" else "Open menu"
+                        prop.className "sm:hidden"
+                        prop.style [ (if isMenuOpen then "black" else "white") |> style.color ]
                     ]
                 ]
             ]
+
+            NavbarMenu [
+                prop.style [ style.paddingTop 0; style.backgroundColor "#F7F4F0" ]
+                menuItems
+                |> List.indexed
+                |> List.map (fun (index, item) ->
+                    NavbarMenuItem [
+                        prop.key $"{item}-{index}"
+                        prop.children [
+                            Link [
+                                color
+                                <| if index = 2 then "foreground"
+                                   else if index = menuItems.Length - 1 then "danger"
+                                   else "foreground"
+                                prop.className "w-full"
+                                prop.style [
+                                    style.fontFamily "the-seasons"
+                                    length.perc 320 |> style.fontSize
```

Comment:
Over 100%? You sure? Maybe this should be a pt size considering it's a font.

Reply:
I had no idea what I was doing...I just tweaked the number until the size looked right. :)

Reply:
Fair enough. Probably good to replace this with a unit you are comfortable working with.

Reply:
I already replaced this one with text-large.